en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-embedded-h2.md

```diff
@@ -5,28 +5,28 @@ The following sections describe how to set up an embedded H2 database to replace
 -   [Setting up the database](#SettingupEmbeddedH2-Settingupthedatabase)
 -   [Setting up the drivers](#SettingupEmbeddedH2-Settingupthedrivers)
 
-!!! warning
-H2 is not recommended in production
+!!! warning 
+    H2 is not recommended in production
 
-The embedded H2 database is NOT recommended in enterprise testing and production environments. It has lower performance, clustering limitations, and can cause file corruption failures. Please use an industry-standard RDBMS such as Oracle, PostgreSQL, MySQL, or MS SQL instead.
+    The embedded H2 database is NOT recommended in enterprise testing and production environments. It has lower performance, clustering limitations, and can cause file corruption failures. Please use an industry-standard RDBMS such as Oracle, PostgreSQL, MySQL, or MS SQL instead.
 
-You can use the embedded H2 database in development environments and as the local registry in a registry mount.
+    You can use the embedded H2 database in development environments and as the local registry in a registry mount.
 
 
 ### Setting up the database
 
 Download and install the H2 database engine on your computer.
 
 !!! info
-For instructions on installing DB2 Express-C, see [H2 installation guide.](http://www.h2database.com/html/quickstart.html)
+    For instructions on installing DB2 Express-C, see [H2 installation guide.](http://www.h2database.com/html/quickstart.html)
 
 
 ### Setting up the drivers
 
-WSO2 currently ships H2 database engine version h2-1.2.140.\* and its related H2 database driver. If you want to use a different H2 database driver, take the following steps:
+WSO2 currently ships H2 database engine version h2_1.4.199.\* and its related H2 database driver. If you want to use a different H2 database driver, take the following steps:
 
 1.  Delete the following H2 database-related JAR file, which is shipped with WSO2 products:
-`<PRODUCT_HOME>/repository/components/plugins/h2-database-engine_1.2.140.wso2v3.jar`
+`<PRODUCT_HOME>/repository/components/plugins/h2_1.4.199.wso2v1.jar`
```

Comment:
Shall we change this <PRODUCT_HOME> to <API-M_HOME> as we are only referring to APIM docs